app/Services/CapacityService.php

```
@@ -0,0 +1,234 @@
+
+    /** Lendable physical copies of a book (the capacity ceiling). */
+    public function totalCopies(int $libroId): int
+    {
+        $stmt = $this->db->prepare(
+            "SELECT COUNT(*) FROM copie
+             WHERE libro_id = ?
+               AND stato NOT IN ('perso','danneggiato','manutenzione','in_restauro','in_trasferimento')"
+        );
+        $stmt->bind_param('i', $libroId);
+        $stmt->execute();
+        $stmt->bind_result($n);
```

PHPMD (2.15.0)

[warning] Avoid using undefined variables such as '$n' which will lead to PHP notices. (undefined)

(UndefinedVariable)